File: web/app/model/pam/Applicant.js
Ext.define('iFlat.model.pam.Applicant', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'applicant.id', mapping: 'id', type: 'string'},
        {name: 'applicant.pbName', mapping: 'pbName', type: 'string'},
        {name: 'applicant.name', mapping: 'name', type: 'string'},
        {name: 'applicant.sex', mapping: 'sex', type: 'string'},
        {name: 'applicant.dept', mapping: 'dept', type: 'string'},
        {name: 'applicant.birth', mapping: 'birth', type: 'date'},
        {name: 'applicant.applyTime', mapping: 'applyTime', type: 'date'},
        {name: 'applicant.isComsomol', mapping: 'isComsomol', type: 'string'},
        {name: 'applicant.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'applicant.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'applicant.createrTime', mapping: 'createrTime', type: 'date'},
    ]
});

File: src/main/java/com/iflat/pam/bean/Applicant.java
package com.iflat.pam.bean;

import java.util.Date;

/**
 * 申请入党人员
 */
public class Applicant {

    private String id;
    private String pbName;
    private String name;
    private String sex;
    private String dept;
    private Date birth;
    private Date applyTime;
    private String isComsomol;

    private String creatorAcc;
    private String creatorName;
    private Date createTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPbName() {
        return pbName;
    }

    public void setPbName(String pbName) {
        this.pbName = pbName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    public String getDept() {
        return dept;
    }

    public void setDept(String dept) {
        this.dept = dept;
    }

    public Date getBirth() {
        return birth;
    }

    public void setBirth(Date birth) {
        this.birth = birth;
    }

    public Date getApplyTime() {
        return applyTime;
    }

    public void setApplyTime(Date applyTime) {
        this.applyTime = applyTime;
    }

    public String getIsComsomol() {
        return isComsomol;
    }

    public void setIsComsomol(String isComsomol) {
        this.isComsomol = isComsomol;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}
